app/notification.go

```diff
@@ -1053,21 +1053,40 @@ func GetExplicitMentions(message string, keywords map[string][]string) *Explicit
 	}
 
 	buf := ""
-	markdown.Inspect(message, func(node interface{}) bool {
-		text, ok := node.(*markdown.Text)
-		if !ok {
-			processText(buf)
-			buf = ""
-			return true
-		}
-		buf += text.Text
-		return false
-	})
+	mentionsEnabledFields := GetMentionsEnabledFields(post)
+	for _, message := range mentionsEnabledFields {
+		markdown.Inspect(message, func(node interface{}) bool {
+			text, ok := node.(*markdown.Text)
+			if !ok {
+				processText(buf)
+				buf = ""
+				return true
+			}
+			buf += text.Text
+			return false
+		})
+	}
 	processText(buf)
 
 	return ret
 }
 
+func GetMentionsEnabledFields(post *model.Post) model.StringArray {
```

Comment:
Can you write a comment for this function please?

Reply:
@theblueskies Added the same